packages/main/cypress/support/commands/TimeSelectionClocks.commands.ts

```diff
@@ -0,0 +1,28 @@
+Cypress.Commands.add("ui5TimeSelectionClocksInnerButton", (timeSelectionClocks, index) => {
+	cy.get(timeSelectionClocks)
+        .shadow()
+        .find("ui5-toggle-spin-button")
+        .as("buttons");
+
+	return cy.get("@buttons")
+        .eq(index)
+        .shadow()
+        .find("button");
+});
+
+Cypress.Commands.add("ui5TimeSelectionClocksIsActiveClock", (timeSelectionClocks, index) => {

```

Comment:
i would switch index from number to "hours" | "minutes" | "seconds" to be more clear